mediawiki/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM php:7-fpm-jessie
+FROM php:7.3-fpm-buster
```

Comment:
Wdyt about leaving this as `php:7-fpm-buster` so we get the latest minor versions?

Reply:
Oh, actually, we can't do that because Mediawiki 1.34 drops support from PHP 7.1 😢 

> Some specific notes for MediaWiki 1.34 upgrades are below:
> * MediaWiki now requires PHP 7.2.9 or above.
>
-- https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/core/+/REL1_34/RELEASE-NOTES-1.34#90